packages/ignite/src/plugins/default/commands/doctor.js

```diff
@@ -0,0 +1,49 @@
+const R = require('ramda')
+const Shell = require('shelljs')
+
+module.exports = async function (context) {
+  try {
```

Comment:
As we chatted offline, `gluegun` can provide a helper mode to not swallow exceptions that you can drive from `--dev`.  Thanks for the suggestion.  Will make that for `0.7.0`.